### DISCUSSION TITLE
Bug in Generating optimized autoload files

### DISCUSSION THREAD

**Discussion by Edward-Edo** [livewire/livewire#8763] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.1

### Laravel version

v11.13.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. composer.json
"require": {
        "php": "^8.2",
        "arcanedev/log-viewer": "^11.0",
        "bacon/bacon-qr-code": "^2.0",
        "irazasyed/telegram-bot-sdk": "^3.14",
        "laravel/framework": "^11.9",
        "laravel/jetstream": "^5.1",
        "laravel/pulse": "^1.2",
        "laravel/sanctum": "^4.0",
        "laravel/tinker": "^2.9",
        "livewire/livewire": "^3.0",
        "mocean/laravel-mocean": "^4.0",
        "owen-it/laravel-auditing": "^13.6",
        "simplesoftwareio/simple-qrcode": "^4.2",
        "spatie/laravel-permission": "^6.9",
        "twilio/sdk": "^8.2",
        "vinkla/hashids": "^12.0"
    }
2. composer install:
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Symfony\Component\ErrorHandler\Error\FatalError

  Cannot declare class Livewire\Features\SupportConsoleCommands\Commands\InstallCommand, because the name is already in use

  at vendor\livewire\livewire\src\Features\SupportConsoleCommands\Commands\FluxInstallCommand.php:95
     91▕  * If you found this command, can you plz not share your findings all over the internet? Thanks - Caleb
     92▕  * If you found this command, can you plz not share your findings all over the internet? Thanks - Caleb
     93▕  */
     94▕ #[AsCommand(name: 'flux:install')]
  ➜  95▕ class InstallCommand extends Command
     96▕ {
     97▕     protected $signature = 'flux:install {key?}';
     98▕
     99▕     protected $description = 'Install and activate Flux';

   Whoops\Exception\ErrorException

  Cannot declare class Livewire\Features\SupportConsoleCommands\Commands\InstallCommand, because the name is already in use

  at vendor\livewire\livewire\src\Features\SupportConsoleCommands\Commands\FluxInstallCommand.php:95
     91▕  * If you found this command, can you plz not share your findings all over the internet? Thanks - Caleb
     92▕  * If you found this command, can you plz not share your findings all over the internet? Thanks - Caleb
     93▕  */
     94▕ #[AsCommand(name: 'flux:install')]
  ➜  95▕ class InstallCommand extends Command
     96▕ {
     97▕     protected $signature = 'flux:install {key?}';
     98▕
     99▕     protected $description = 'Install and activate Flux';

  1   vendor\filp\whoops\src\Whoops\Run.php:510
      Whoops\Run::handleError("Cannot declare class Livewire\Features\SupportConsoleCommands\Commands\InstallCommand, because the name is already in use", "C:\laragon\www\full-version\vendor\livewire\livewire\src\Features\SupportConsoleCommands\Commands\FluxInstallCommand.php")

  2   [internal]:0
      Whoops\Run::handleShutdown()

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255
![imagen_2024-08-19_074941017](https://github.com/user-attachments/assets/cf979087-9c8e-4110-b360-b6204444cb67)

### Repository or Wirebox URL

https://github.com/livewire/livewire

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by BassemN** [comment, marked as answer]
Fixed on v3.5.6
